observatory-platform/observatory/platform/api/app/openapi.yml

```diff
@@ -0,0 +1,142 @@
+swagger: '2.0'
+info:
+  title: Elasticsearch API
+  description: 'Query data from elasticsearch indexes'
+  version: 0.1.9
+#securityDefinitions:
+#  APIKeyHeader:
+#    type: apiKey
+#    in: header
+#    name: X-API-Key
+#    x-apikeyInfoFunc: server.basic_auth
+#  APIKeyQueryParam:
+#    type: apiKey
+#    in: query
+#    name: api_key
+#    x-apikeyInfoFunc: server.basic_auth
+#security:
+#  - APIKeyHeader: []
+#  - APIKeyQueryParam: []
+paths:
```

Comment:
Just commenting here to document a conversation I just had with Jamie. While it would be good to get this first version merged without too much detailed analysis of the paths, we do need to setup ourselves up for future changes. I suggest (with exact details to be worked through, following best practice examples after some research) prefixing any method behind something roughly equivalent to url.com/version/0.1/methodName

This will allow us to maintain older versions of the API and put in place depreciation schedules, while also addressing new requirements as they come in.